src/lib.rs

```diff
@@ -400,21 +400,21 @@ use futures::channel::oneshot;
 use once_cell::sync::OnceCell;
 use pyo3::{call::PyCallArgs, prelude::*, types::PyDict};
 
-static ASYNCIO: OnceCell<PyObject> = OnceCell::new();
-static CONTEXTVARS: OnceCell<PyObject> = OnceCell::new();
-static ENSURE_FUTURE: OnceCell<PyObject> = OnceCell::new();
-static GET_RUNNING_LOOP: OnceCell<PyObject> = OnceCell::new();
+static ASYNCIO: OnceCell<Py<PyAny>> = OnceCell::new();
+static CONTEXTVARS: OnceCell<Py<PyAny>> = OnceCell::new();
+static ENSURE_FUTURE: OnceCell<Py<PyAny>> = OnceCell::new();
+static GET_RUNNING_LOOP: OnceCell<Py<PyAny>> = OnceCell::new();
```

Comment:
I think these should probably be updated to use the new `PyOnceLock` (and `PyOnceLock::import`, potentially).

Reply:
Ah yes, these ones which don't need Python can probably now be replaced with `std::sync::OnceLock`, which is available after we bump MSRV.

Reply:
I was able to remove almost all uses of `once_cell`, except for 
https://github.com/PyO3/pyo3-async-runtimes/blob/5e6c5b7667c63ea374bda93934ac2429cac2ad2c/src/tokio.rs#L68
since it looks like `LazyLock` wasn't added to the stdlib until an MSRV of 1.80.